consensus/ibft/src/test/java/org/hyperledger/besu/consensus/ibft/IbftExtraDataTest.java

```diff
@@ -464,6 +464,14 @@ public void incorrectVoteTypeThrowsException() {
         .isInstanceOf(RLPException.class);
   }
 
+  @Test
+  public void emptyExtraDataThrowsException() {
+    final BytesValue bufferToInject = BytesValue.EMPTY;
+
+    assertThatThrownBy(() -> IbftExtraData.decodeRaw(bufferToInject))
+        .isInstanceOf(IllegalArgumentException.class);
```

Comment:
To ensure the exception returned is from the intended throw (although there is only one condition here, there could be many more `IllegalArgumentException` conditions in the future), it might also be worth asserting the message is that expected? 

```suggestion
        .isInstanceOf(IllegalArgumentException.class) 
        .hasMessage("Invalid BytesValue supplied - Ibft Extra Data required.");
```

Reply:
I totally agree, PR updated.